java/registry/src/main/java/io/opensaber/registry/sink/DBShard.java

```diff
@@ -17,16 +20,19 @@
 	@Autowired
 	DBConnectionInfoMgr dBConnectionInfoMgr;
 	
-	public DatabaseProvider getInstance(String shardId){
+	@Autowired
+	AdvisoryLoader advisoryLoader;
+		
+	public DatabaseProvider getInstance(String property, String value) throws IOException{
```

Comment:
I thought the DBShard just manages the connection. It does not need to know the attribute, value details. Given a shardId or connectionInfo, it merely gives initializes a provider. All this must either go to a separate class or this class be re-invented.
Treat DBShard as one that can just give a 'graph' instance, when you supply a connectionInfo.
Treat Advisor as one that merely advises the connectionInfo to be used for a given attribute value.
Treat X (to be invented) as one that can handle both the above classes and give you 'graph' instance.

Reply:
Done.